jobs/stylegan3/job_stylegan3-t_gen_images.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#SBATCH --job-name=stylegan3-t-gen-images
+#SBATCH --time 10-00:00
+#SBATCH --nodes=1
+#SBATCH --gpus-per-node=v100l:1
+#SBATCH --mem=128G
+#SBATCH --ntasks-per-node=4
+#SBATCH --cpus-per-task=4
+#SBATCH --account=def-whkchun
+#SBATCH --output=%x.o%j
+ 
+pip install --no-index --upgrade pip
```

Comment:
Nothing to change right now, but a comment:  The sbatch scripts for StyleGAN2-ADA used an environment previously built in my home directory.  My thinking was to _not_ use training time to build the environment at the start of each job, since that environment was fixed for all of the training and image generation.

On the other hand, those scripts rely on an environment in my home directory, which is accessible only to me.  With the introduction of the `s2_bliss` shared folder, an environment can be shared.

Anyway, something to consider in the future.  I wouldn't change anything now.